packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -317,6 +318,25 @@ export const FLAGS: Record<string, FlagInfo> = {
     compatibilityWithOldBehaviorMd: 'You can pass `desiredCount: 1` explicitly, but you should never need this.',
   },
 
+  [ECS_PATTERNS_SMART_DEFAULT_OPEN_LISTENER]: {
+    type: FlagType.ApiDefault,
+    summary: 'Use smart defaults for openListener when custom security groups are provided',
```

Comment:
And if it is renamed, "smart defaults" should be replaced by a more descriptive term.